basedirs/info.go

```diff
@@ -89,6 +96,10 @@ func countFromFullBucketScan(tx *bolt.Tx, bucketName string,
 	return count, sliceLen
 }
 
+func checkAgeOfKeyIsAll(key []byte) bool {
+	return bytes.Split(key, []byte{bucketKeySeparatorByte})[2][0] == ageToByteSlice(summary.DGUTAgeAll)[0]
```

Comment:
Again, it might be better to have bucketKeySeparatorByte as a byte-slice to avoid the allocation on every call.

Reply:
Added global for this since we treat it as a constant and don't need to change it in tests either.